engine.py

```diff
@@ -191,6 +191,13 @@ def show_panel(self, panel_id, title, bundle, widget_class, *args, **kwargs):
 
         :returns: the created widget_class instance
         """
+        mari_version = mari.app.version()
+        if mari_version.major() < 4:
+            raise AttributeError("This version of Mari ({}.{}v{}) may have palette resize issues. "
```

Comment:
AttributeError because that is what happens when show_panel is not defined here.